index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="./assets/css/styles.css">
+</head>
+
+<body>
+  <header id="header">
+    <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png"
+      alt="original trombones logo">
+    <nav id="nav-bar">
+      <ul>
+        <li><a class="nav-link" href="#features">Features</a></li>
+        <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+        <li><a class="nav-link" href="#pricing">Pricing</a></li>
+      </ul>
+    </nav>
+  </header>
+  <div class="container">
```

Comment:
We can improve the semantics of your code. Please change the div container to be a section tag. It is a good practice to follow the sequence section>article>div

Reply:
This is a great review comment 💪🏻. It sounds polite and friendly 😃 and presents a reason why changes should be made. Well done @Leboroz 👏🏻!